packages/termux-tools/termux.1.md.in

```diff
@@ -316,6 +325,12 @@ or have restarted the application.
     If the property is not set, then application will choose colors
     accordingly to the current system theme.
 
+'allow-external-apps'
```

Comment:
This will also be required for termux content provider, including `termux-open` in next version.